modules/pam_faillock/Makefile.am

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2005, 2006, 2007, 2009 Thorsten Kukuk <kukuk@thkukuk.de>
+# Copyright (c) 2008, 2018, 2020 Red Hat, Inc.
+# Copyright (c) 2010 Tomas Mraz <tmraz@redhat.com>
+#
+
+CLEANFILES = *~
+MAINTAINERCLEANFILES = $(MANS) README
+
+EXTRA_DIST = README $(MANS) $(XMLS) faillock.conf tst-pam_faillock
```

Comment:
This EXTRA_DIST can be simplified to
```EXTRA_DIST = $(DATA) $(MANS) $(XMLS) $(TESTS)```
similar to recent cleanups in other makefiles.